doc/source/release/1.16.0-notes.rst

```diff
@@ -302,6 +302,8 @@ Build and packaging related changes
 *************
 Other changes
 *************
+- A new accompanying release of ``scipy-stubs`` (``v1.16.0.0``) is
+  available.
```

Comment:
Perhaps some links might help:

```suggestion
- A new accompanying release of `scipy-stubs
  <https://github.com/scipy/scipy-stubs>`_ (`v1.16.0.0 
  <https://github.com/scipy/scipy-stubs/releases/tag/v1.16.0.0>`_) is available.
```

I wasn't able to figure out to keep the inline code style though 🤔